builder-base/Makefile

```diff
@@ -36,6 +36,8 @@ images:
 		--local dockerfile=./ \
 		--local context=. \
 		--output type=image,oci-mediatypes=true,name=$(IMAGE),push=true
+	cd ../../eks-distro-pr-bot/eks-distro-build-tooling
```

Comment:
Just curious, is there a presubmit that tests this in a dry run mode or something? Also change the path like below comment

Reply:
There isn't, but I could change the eks-distro-presubmit to have similar changes to the postsubmit. Remind me, do we have the secrets mounted on the presubmit cluster too?

Reply:
We may not but it is simple to add them. If you think there is a way for us to test some of this logic in a presubmit, then it would only be a plus for us to catch issues right?

Reply:
We actually should not mount the secrets to presubmits if possible. Is there a way to dry run, or just not call anything that requires the github client in the presubmit?